crates/core_simd/src/masks.rs

```diff
@@ -88,6 +88,11 @@ impl_element! { isize }
 /// The layout of this type is unspecified, and may change between platforms
 /// and/or Rust versions, and code should not assume that it is equivalent to
 /// `[T; LANES]`.
+///
+/// For a type with size guaranteed equal to `[T; LANES]`, use
```

Comment:
```suggestion
/// For a type with size guaranteed equal-or-greater-than `[T; LANES]`, use
```

Reply:
```suggestion
/// For a type equivalent to `[T; LANES]` (with possible trailing padding for alignment purposes), use
```
maybe equivalent is too strong? ideas?

Reply:
> ```
> /// For a type with size guaranteed equal-or-greater-than `[T; LANES]`, use
> ```

imho that loses most meaning, e.g. `[u8; 5000]` fits but is unlikely to be desired.

Reply:
"equivalent" is indeed too strong.